porting/3_4_Halium_7-1_porting.rst

```diff
@@ -0,0 +1,117 @@
+Configuring Ubuntu Touch (7.1)
+==============================
```

Comment:
this file has three heading layers:

Configuring UT
+ Configuring and eable remaining device functionality
++ Udev rules
++ Display scaling
++ Apply apparmor kernel patches
++ Sound configuration
+ Further configuration

I propose to move udev, scaling, apparmor and sound one level up and split the "Configure and enable remaining.." section into a) a general intro and a new "overlay" section. Like this:

Configuring UT 
"In the previous section you adjusted the udev settings .... overlay files, as described below."
+ Overlay files 
"Overlay files allow you to deploy configurations that are specific to your port in a place that normally belongs to the rootfs. Note that as a rule of thumb ... "
+ Udev rules
+ Display scaling
+ Apply apparmor kernel patches
+ Sound configuration
+ Further configuration

Reply:
I agree. I wonder if the same should be done for two of the other files. Will try...